### PR TITLE
Seeds for data added

### PR DESCRIPTION
**Pull request by Chulstro**
# SOM-Timer BE Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
`This adds a seed file that can be run with 'flask db_seed'`

## Type of change
- [x] Other, Please explain:
`Seeding command rebuffed`

## Has This Been Tested?
- [x] No
- [ ] Yes

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`No official in-line tests but tested locally to see that seeds are in fact created`

### CODE REVIEW

**Review by jkwest-93 — APPROVED** [review]
Yay content!!! Looking forward to checking these out when we get the endpoint hooked up.

**Review by rwilliams659 — APPROVED** [review]
Thanks for seeding this data!

File: app.py [hunk 1]
```diff
@@ -3,20 +3,62 @@
...
+
+    #SOMATIC
+    Seeds.append(Exercise(url='https://www.youtube.com/watch?v=dsmfIAyiois', duration='5:00', category='SOMATIC'))
+    Seeds.append(Exercise(url='https://soundcloud.com/ucsdmindfulness/mpeak-5-min-body-scan-by-pete-kirchmer', duration='5:00', category='SOMATIC'))
+    Seeds.append(Exercise(url='https://www.youtube.com/watch?v=Eum8DTvsRkI', duration='7:00', category='SOMATIC'))
+    Seeds.append(Exercise(url='https://www.youtube.com/watch?v=1saZqSr2zGM', duration='7:00', category='SOMATIC'))
+    Seeds.append(Exercise(url='https://soundcloud.com/ucsdmindfulness/10-min-body-scan-by-christy-cassisa?in=ucsdmindfulness/sets/short-meditation-sessions', duration='10:00', category='SOMATIC'))
+    Seeds.append(Exercise(url='https://www.youtube.com/watch?v=zsCVqFr6j1g&feature=emb_logo', duration='10:00', category='SOMATIC'))
+
+    #Default Timer
```
Comment (sienna-kopf) [inline app.py:57]: So sweet. fantastic having all of these seeds in here.

File: app.py [hunk 2]
```diff
@@ -3,20 +3,62 @@
...
 db = SQLAlchemy(app)
 
 @app.cli.command('db_seed')
 def db_seed():
-    exercise1 = Exercise(url='https://www.youtube.com/watch?v=W5wqniA4MMc&ab_channel=EssentialSomatics', duration='10:00', category='MEDITATION')
-    exercise2 = Exercise(url='https://www.youtube.com/watch?v=_0xZ3iOswYM', duration='10:00', category='MOVEMENT')
 
-    timer1 = Timer(work_interval='25:00', rest_interval='5:00')
+    #Seed Array
+    Seeds = []
```
Comment (sienna-kopf) [inline app.py:16]: Clever use of an array here to send them all back in the right format!

**Review by sienna-kopf — APPROVED** [review]

**Pull request merged**